<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivedSignalCfg</class>
 <widget class="QWidget" name="ActivedSignalCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>激活模块配置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CgeneralControlSoftware.qrc">
    <normaloff>:/新前缀/img/logo.png</normaloff>:/新前缀/img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
font: 12pt &quot;SimSun&quot;;
}
QPushButton{
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	
	font: 11pt &quot;SimSun&quot;;
	border: 1px solid rgb(150,150,150);
	
}

QPushButton:hover {
	background-color: rgb(200, 200, 200);
}

 QPushButton:pressed
{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>41</y>
     <width>428</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,2">
    <item>
     <widget class="QLabel" name="labelCfgChannel">
      <property name="text">
       <string>配置通道：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combCfgChannel">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>28</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>第1路激活信号通道</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第2路激活信号通道</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第3路激活信号通道</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCfgApplyToAll">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>应用到所有</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>79</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>电压：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinBoxVoltage">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="minimum">
       <double>25.000000000000000</double>
      </property>
      <property name="maximum">
       <double>35.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>电流：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinBoxCurrent">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>441</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,0,3,0,3,0,3">
    <item>
     <widget class="QPushButton" name="btnOpen">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveAs">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>另存为</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CgeneralControlSoftware.qrc"/>
 </resources>
 <connections/>
</ui>
